src/cmake/config/win-vs.txt

```diff
@@ -154,18 +154,6 @@ if (USE_AVX2)
         /arch:AVX2                          # Advanced Vector Extensions 2
     )
 endif ()
-if (USE_FMA)
```

Comment:
Correction, it looks like F16C is supported at least by my CPU (Haswell) :) Not sure to which instruction set it belongs, but it looks like you were right: the first CPU with AVX2 support is Haswell, and it also supports FMA and F16C, so let's go with that.